File: Java/Data Structures & Algorithms Module - Java/Revision/BinarySearchTree2/Node.java
/*
Node.java
Billy Cussen
25/01/2021
*/
public class Node<T>{
    
    T element;
    Node<T> left;
    Node<T> right;

    public Node(T element){
        this.element = element;
        this.left = null;
        this.right = null;
    }

    @Override
    public String toString(){
        String leftNode = this.left == null ? "NULL" : this.left.element.toString();
        String rightNode = this.right == null ? "NULL" : this.right.element.toString();
        return "Element: "+this.element+", Left: "+leftNode+", Right: "+rightNode;
    }

}

File: Python/Data Structures & Algorithms Module - Python/Revision/BST1/Node.py
"""
Node.py
Billy Cussen
26/01/2021
"""

class Node:

    def __init__(self, element):
        self.element = element
        self.left = None
        self.right = None

    def toString(self):
        leftNode = "NULL" if self.left == None else self.left.element
        rightNode = "NULL" if self.right == None else self.right.element
        return "Element: "+str(self.element)+", Left: "+str(leftNode)+", Right: "+str(rightNode)